README.md

```diff
@@ -14,7 +14,7 @@
         <img src="https://img.shields.io/github/tag/sulu/sulu.svg" alt="GitHub tag (latest SemVer)">
     </a>
     <a href="https://github.com/sulu/sulu/actions" target="_blank">
-        <img src="https://img.shields.io/github/workflow/status/sulu/sulu/Test%20application.svg?label=test-workflow" alt="Test workflow status">
+        <img src="https://img.shields.io/github/actions/workflow/status/sulu/sulu/test-application.yaml?branch=2.5" alt="Test workflow status">
```

Comment:
I would remove the `branch=2.5` it something which will be just forgotten to be updated. Like beneath the `https://github.com/sulu/sulu/commits/2.x` which I would just link to `https://github.com/sulu/sulu/commits`.